src/main/java/net/hydromatic/lookml/Spaces.java

```diff
@@ -19,15 +19,17 @@
 package net.hydromatic.lookml;
 
 /**
- * LookML main class.
+ * Utilities for writing spaces.
  */
-public class Lookml {
-  private Lookml() {
+class Spaces {
+  private Spaces() {
   }
 
-  public static void main() {
-    System.out.println("Hello, world!");
+  static void append(StringBuilder b, int indent) {
```

Comment:
Since most indents are of the same length, it could be worth considering a Spaces.append(String) to remove the for..loop

Reply:
Logged #19 for this